chapter-07/recipe-05/README.md

```diff
@@ -3,3 +3,4 @@
 Abstract to be written ...
 
 - [example](example/)
+- [fortran-example](fortran-example/)
```

Comment:
Same comment as above the `fortran-example` was removed, but the empty folder still lies around, confusing `generate-readmes.py`.